koocook_core/models/nutrition.py

```diff
@@ -9,12 +8,6 @@
 __all__ = ['MetaIngredient', 'RecipeIngredient']
 
 
-@enum.unique
-class NutrientType(enum.Enum):
-    CARBOHYDRATE = 'carbohydrates'
-    ...
```

Comment:
Why does this line have an ellipsis? Does it mean something?

Reply:
It's a deletion? It's in red. Not green.

Reply:
OK